presto-raptorx/src/main/java/com/facebook/presto/raptorx/metadata/CommitCleaner.java

```diff
@@ -175,6 +195,41 @@ public synchronized void removeOldCommits()
         }
     }
 
+    // TODO(taozhao) For workers, shardDao doesn't have to be a list, each one only deals with its own node_id. May write a separate Cleaner later.
```

Comment:
Yes. If you want to to cooperative clean up for organization garbage, it would be better to be in the different class. 

But one step back, is there any concern using a centralized cleaner within the coordinator?

Reply:
I think in coordinator is fine for now. That mysql request is only 1 or 2 milliseconds. 

Meanwhile changing CommitCleaner is super serious. I will keep the way now.